### ISSUE TITLE
FoC launcher not detectiong mods if RaW is not installed

### ISSUE THREAD

**Issue by BlackburnUTG** [AnakinSklavenwalker/FoC-Mod-Launcher#15]
After I unsubscribed from RaW - FoC launcher can't see any other mods,
This issue occur not only for  me, but for many other people I found on eaw discord servers

**Comment by AnakinRaW** [comment]
Still an Issue?
What are the other telling? Any more details on this?

**Comment by BlackburnUTG** [comment]
yeah, I still cannot see any mods, if I don't have RaW installed. Don't know what kind of extra info I can share with you. 
I tested with my friends - they have the same issue

**Comment by AnakinRaW** [comment]
ok. I can't really reproduce it.

Maybe try launching the launcher while pressing the SHIFT key. that should reset the launcher. If the error still occurs, here are some questions i have.

Which game do you have? Steam, Disk, GOG or Origin? Do you have multiple games installed? Does the detected game of the launcher match with the game you intend it to find? Which mods do you have installed? Where are these mods installed? It the error resolved if you install RaW again?

**Comment by BlackburnUTG** [comment]
Launching with SHIFT key doesn't help.
I have steam version, the game installed on my C drive ssd.
I have Aotr, RaW, and raw submod by Myth
here you can see when I was deleting RaW - other mods dissaperes, they come back when raw restored, and raw stays when others gets deleted 
https://www.youtube.com/watch?v=VvD8NChhqRw

**Comment by messnerfe** [comment]
I can confirm this, too. 

I was cleaning out a lot of mods for disk space, and since then I see only the base games (neither workshop mods, nor non-workshop mods from `corruption/Mods/`). After seeing this thread, I just reinstalled RaW, and now everything is back again.

I've narrowed it down to RaW's modinfo.json file. If that's renamed or completely missing, no mods are detected when starting the launcher. If the file exists, but is completely empty, also no mods in the launcher.

As soon as the file contains at least an empty object `{}`, other mods get detected, but RaW itself is still missing. 
To get RaW to be detected, at least the `name` field needs to be present in the modinfo file. Apparently if that's not there, the whole mod is skipped.

I hope this can help narrow it down, if you need any more info, I'll be happy to provide :)
